packages/desktop-client/src/components/transactions/SelectedTransactionsButton.jsx

```diff
@@ -147,7 +149,7 @@ export function SelectedTransactionsButton({
 
   return (
     <SelectedItemsButton
-      name="transactions"
+      name={count => i18n.t('{{count}} transactions', { count })}
```

Comment:
nit: I noticed that in some places we use `i18n.t` and some places we destructure to use `t` directly. IMO we should settle on one convention for consistency.

Reply:
Indeed, you have a keen eye. I did not destructure here because we already use `t` for transactions in this component and I didn't want to have a potential for confusion.